### PR TITLE
"The Different Biomes" by winniepee0

### PR DESCRIPTION
**Pull request by winniepee**

### CODE REVIEW

**Comment by AnyBlue** [comment]
Looks nice. But I recommend to change aspen on pine because in Minetest no aspen on snow...

**Comment by olliy1or** [comment]
This PR is making changes to another map, submitting a map should only add the maps .mts .conf and a screenshot nothing else

File: abandoned_isles/map.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@ rotation = x
 h = 140
 team.2 = blue
 name = Abandoned Isles
+hint = Find a bunch of mese blocks!
```
Comment (Panquesito7) [inline abandoned_isles/map.conf:11]: Please revert this change.

File: map.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+team.1.pos = (87,-26,100)
```
Comment (Panquesito7) [inline map.conf:1]: Please remove this file. There's a `map.conf` for Different Biomes map already.

**Comment by JostP** [comment]
You should create new branch from master and then make a PR.

**Comment by winniepee** [comment]
Ok
